### PR TITLE
[New] Human Task Resource API

### PR DESCRIPTION
**Pull request by Mkatari3700**
Changes to address the following
1. Added new **HumanTaskResourceAPI** 
2. Added test case sample for human task (one get call)
3. Added constants file

### CODE REVIEW

File: Conductor/Api/IHumanTaskResourceApi.cs [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+        /// <summary>
+        /// API for backpopulating index data
+        /// </summary>
+        /// <remarks>
+        /// 
+        /// </remarks>
+        /// <exception cref="ApiException">Thrown when fails to make API call</exception>
+        /// <param name="_100"></param>
+        /// <returns>Dictionary&lt;string, Object&gt;</returns>
+        Dictionary<string, Object> BackPopulateFullTextIndex(int? _100);
```
Comment (v1r3n) [inline Conductor/Api/IHumanTaskResourceApi.cs:36]: we can remove this API -- this is only for some maintenance

File: Conductor/Api/IHumanTaskResourceApi.cs [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+        /// <summary>
+        /// Get a task
+        /// </summary>
+        /// <remarks>
+        /// 
+        /// </remarks>
+        /// <exception cref="ApiException">Thrown when fails to make API call</exception>
+        /// <param name="taskId"></param>
+        /// <returns>HumanTaskEntry</returns>
+        HumanTaskEntry GetTask1(string taskId);
```
Comment (v1r3n) [inline Conductor/Api/IHumanTaskResourceApi.cs:94]: Can we rename this as `GetTask`

File: Conductor/Api/IHumanTaskResourceApi.cs [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+        /// <summary>
+        /// API for backpopulating index data
+        /// </summary>
+        /// <remarks>
+        /// 
+        /// </remarks>
+        /// <exception cref="ApiException">Thrown when fails to make API call</exception>
+        /// <param name="_100"></param>
+        /// <returns>Task of Dictionary&lt;string, Object&gt;</returns>
+        System.Threading.Tasks.Task<Dictionary<string, Object>> BackPopulateFullTextIndexAsync(int? _100);
```
Comment (v1r3n) [inline Conductor/Api/IHumanTaskResourceApi.cs:263]: Let's remove this method.

**Review by v1r3n — CHANGES_REQUESTED** [review]
some minor comments with method names and internal methods that we should not expose in the sdk.

File: Conductor/Api/IHumanTaskResourceApi.cs (hunk 2 above)
Comment (Mkatari3700) [reply Conductor/Api/IHumanTaskResourceApi.cs]: Renamed

File: Conductor/Api/IHumanTaskResourceApi.cs (hunk 3 above)
Comment (Mkatari3700) [reply Conductor/Api/IHumanTaskResourceApi.cs]: Removed

File: Conductor/Api/IHumanTaskResourceApi.cs (hunk 1 above)
Comment (Mkatari3700) [reply Conductor/Api/IHumanTaskResourceApi.cs]: removed

**Review by v1r3n — APPROVED** [review]

**Pull request merged**